### PR TITLE
Fixes #1996 - update contribute.md

### LINKED ISSUE #1966: Fix navigation
Looks like the navigation got messed up during a merge.

### LINKED ISSUE #1996: Update Contributing.md
This is about updating the Contributing.md with the suggested new process for labels and feature requests.

### PR DESCRIPTION
**Pull request by zoepage**
r? @miketaylr

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ If you're using webcompat.com and something is confusing, broken, or you think i
 * Search the [issues list](https://github.com/webcompat/webcompat.com/issues) for existing similar issues.  Consider adding to an existing issue if you find one.
 * Choose a descriptive title.
 * Provide a test, snippet of code or screenshot that illustrates the problem. This small gesture goes a long way towards getting a speedy fix.
+* Please always add a `type` label to the issue. If it fits, you also can add a `scope` and / or `language` label. Feel free to take a look at the [label list](https://github.com/webcompat/webcompat.com/labels) first.
```
Comment (miketaylr) [inline CONTRIBUTING.md]: One note: non-collaborators won't be able to add labels to the issue. Maybe we can add some language here to reflect that?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,22 +38,23 @@ One way to contribute is to triage issues. This could be as simple as confirming
...
 
-You can request a new feature by [submitting an issue](#filing-an-issue) to our repo.  If you would like to implement a new feature then consider what kind of change it is:
+You can request a new feature by [submitting an issue](#filing-an-issue) to our repo. If you would like to implement a new feature then consider what kind of change it is:
 
-* **Major Changes** that you wish to contribute to the project should be discussed first in an issue or irc so that we can better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
+* **Major Changes** that you wish to contribute to the project should be discussed first in an issue so that we can better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
 * **Small Changes** can be crafted and submitted as a Pull Request.
 
 
+
```
Comment (miketaylr) [inline CONTRIBUTING.md]: nit: Dunno if this space is meaningful (but also, I don't think it matters...)

File: CONTRIBUTING.md (hunk 1 above)
Comment (zoepage) [reply CONTRIBUTING.md]: That's a good point. 👍

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**